packages/compass-preferences-model/src/preferences-schema.tsx

```diff
@@ -810,17 +810,16 @@ export const storedUserPreferencesProps: Required<{
       .default('atlas'),
     type: 'string',
   },
-  optInDataExplorerGenAIFeatures: {
+  optInGenAIFeatures: {
     ui: true,
     cli: false,
     global: false,
     description: {
-      short: 'User Opt-in for Data Explorer Gen AI Features',
+      short: 'User or Client Opt-in for Gen AI Features',
     },
-    validator: z.boolean().default(true),
+    validator: z.boolean().default(false),
```

Comment:
correct me if not so but this change shouldn't have any side effects on DE. The state of this was being always being overriden by the API request and we'd likely not have wanted this to be true anyhow.